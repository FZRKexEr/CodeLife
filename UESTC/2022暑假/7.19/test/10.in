10
wljqrcxlsppcqpsasqgubxvwrplshknxcbkmjnlfvwejyafzwdkskovefrmionbakqtowyhhyftsrhdlpzufqcyqkwdmlzgwqzfc
10
nhasgigsyyqertykhmmkcugbnpafydtwzwmiwrmajgvbuiibixtqjfqaupwzecdfnrugtqnvltbkxrwzsxhfwqvnwxyjczdmnafr
8
wsohlisvfyxdwvuecfvhteopimxgsmnxvbvpzlmtmjojouneejhtuxckzlnyehivyyixomrxmwszeeplxiznzgkuxtymdzavtxfx
3
yevozmrxbhldqbdbjdoawcbysxtwzsdbenopkojbkhgjvhetxnutbdvtarcomhngczlkivcjfvitohrafozojnoaoncnpgtudsab
7
zutxdiydbpvnlstxpbzhtueatwkxzrshvjeicfpteoktsbooshuynaawvhxazivvawmyclndoyaojwomlcyihyqnvbulhoeexlst
4
tkwapfibaktqhmghnmyxshweonvdwffhnzfvjdapoiombfnjtkqrkinfogsrcbzmxmqruuvugkursnxqtwwdlrxlqkuxyumuiavm
7
tawwcfvqlmrenazehafswxouahpvosskeumobxviuljjphfkxrsrqvbqivylfogorcqocxxwolbvliceyulwpfizjsazpgyfjemh
6
mrijxjgijxejutvwmgbkzexjiyacokxewfgrsgabxelixubskqytvvhhhoogiyeoriimabexqiyaaemrwcsuxpyinfgjpubnlkqt
8
jpdithaodnumpgftmunsczdopmdouphgbaghmgiycijjjtpfuikrpiomzndeyhjmkafivqhzbfzmvefxlsfnavagbsyaojjjersv
9
kdnzkcsfaxhhrxsknmueikyjjedpfdajhgweyloqvltitwnnxjgokigexvwlpyufzyxrfwqiefkjfbmyebcsafjofitbtymnaone
9
