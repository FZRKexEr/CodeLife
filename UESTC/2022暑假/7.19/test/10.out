1024
4096
603979768
16384
33554432
16384
65536
4096
2048
2048
